R/dash.R

```diff
@@ -293,7 +293,7 @@ Dash <- R6::R6Class(
  
         # inspect the output_value to determine whether any outputs have no_update
         # objects within them; these should not be updated
-        if (class(output_value) == "no_update") {
+        if (length(output_value) == 1 && class(output_value) == "no_update") {
```

Comment:
Great! What's the case where this extra condition is necessary? And which test exercises that condition? 😈

Reply:
> Great! What's the case where this extra condition is necessary? And which test exercises that condition? 😈

:joy: I saw that one coming!

- the case is any `output_value` which is a vector or list with two or more elements (and for which `if` would return this warning since the function is not vectorized)
- I could add a button which returns a vector and then send a click to see if the warning doesn’t arise; is that sufficient?

Reply:
> I could add a button which returns a vector and then send a click to see if the warning doesn’t arise; is that sufficient?

perfect.

Reply:
fixed in 8f6dafd